app/src/components/governance/PageProposal.vue

```diff
@@ -170,7 +170,11 @@
       <ModalVoteSubstrate
         v-if="currentNetwork.network_type === `polkadot`"
         ref="modalVote"
-        :proposal-id="proposalId"
+        :proposal-id="
+          currentNetwork.network_type === `polkadot`
```

Comment:
why? this should be always the same. cosmos should also provide the proposalId, right?